tests/prague/eip7702_set_code_tx/test_gas.py

```diff
@@ -998,3 +998,43 @@ def test_self_set_code_cost(
             ),
         },
     )
+
+
+def test_call_to_pre_authorized_oog(
+    state_test: StateTestFiller,
+    pre: Alloc,
+):
```

Comment:
```suggestion
@pytest.mark.with_all_call_opcodes()
def test_call_to_pre_authorized_oog(
    state_test: StateTestFiller,
    pre: Alloc,
    fork: Fork,
    call_opcode: Op,
):
```

To run this same test with all call opcodes.